replica/replica.go

```diff
@@ -776,7 +776,10 @@ func (r *Replica) createNewHead(oldHead, parent, created string) (types.DiffDisk
 	}
 
 	if _, err := os.Stat(r.diskPath(newHeadName)); err == nil {
```

Comment:
Can we also have a check for the space consumed by the file over here. The space consumed by the file should be zero. If not, we can sleep and fatal out.

Reply:
done